designer/client/conditions/inline-condition-helpers.js

```diff
@@ -1,8 +1,8 @@
-export async function storeConditionIfNecessary (data, conditions) {
+async function storeConditionIfNecessary (data, conditions) {
   let condition
   if (conditions && conditions.hasConditions) {
     condition = await data.getId()
```

Comment:
I must've missed this. feel free to resolve in another PR. data.getId() looks like it's getting the id of the data object. maybe this should be renamed generateId?

Reply:
I'll resolve this another time, but yes, good point!